Foundation/NSCalendar.swift

```diff
@@ -163,7 +163,10 @@ public class NSCalendar : NSObject, NSCopying, NSSecureCoding {
     }
     
     public func copyWithZone(_ zone: NSZone) -> AnyObject {
-        NSUnimplemented()
+        let copy = NSCalendar(calendarIdentifier: calendarIdentifier)!
+        copy.locale = locale
+        copy.timeZone = timeZone
+        return copy
```

Comment:
@parkera 

CFCalendarGetFirstWeekday, CFCalendarGetMinimumDaysInFirstWeek which are called to populate firstWeekday, minimumDaysInFirstWeek fields derives it from 

> _cfobject->cal

 initialised based on identifier, locale and timezone. So, it is not required to set these explicitly in copyWithZone()